packages/react/modules/components/configure-adapter/configure-adapter.tsx

```diff
@@ -244,20 +249,20 @@ export default class ConfigureAdapter extends React.PureComponent<
         )}
       >
         {(() => {
-          const isAdapterReady: boolean = this.props.adapter.getIsReady();
+          const isAdapterReady = this.props.adapter.getIsReady();
 
           if (isAdapterReady) {
             if (typeof this.props.render === 'function')
               return this.props.render();
           }
 
-          if (typeof this.props.children === 'function')
+          if (isFunctionChildren(this.props.children))
```

Comment:
This is a consequence of the "correct" union type, as TS cannot infer the correct type

<img width="729" alt="image" src="https://user-images.githubusercontent.com/1110551/71520042-83e3e480-28ba-11ea-9a99-cb880204fe48.png">